src/Sql/dbo/Stored Procedures/ProviderUser_ReadByProviderId.sql

```diff
@@ -10,4 +11,5 @@ BEGIN
         [dbo].[ProviderUserView]
     WHERE
         [ProviderId] = @ProviderId
+    AND (@Type IS NULL OR [Type] = @Type)
```

Comment:
`OR` operator in SQL is slow (impacts query execution plan), use `[Type] = COALESCE(@Type, [Type])` instead.

Reply:
I copied it from orgs, do we want to update those sprocs as well?

Reply:
No, let's keep the same pattern in place if it's already established in Orgs